src/index.css

```diff
@@ -29,16 +29,12 @@ code {
 .outer-container {
   border: 2px solid #b3b4b8;
   width: 324px;
-  height: 250px;
-  margin: 40px auto 350px auto;
+  height: 300px;
+  margin: 40px 100px auto 1400px;
```

Comment:
- [ ] Good job so far but kindly make sure your Calculator page displays correctly, instead of `1400px` of margin to the left you can decrease it to `1100px`

Your design

<img width="1081" alt="Screenshot 2023-09-20 at 14 07 58" src="https://github.com/newtonah0/Math-magicians/assets/74501523/3d71b415-17ac-45f2-a114-3b4793dbf889">